README.md

```diff
@@ -0,0 +1,65 @@
+# gulp-eval
+[![NPM version](http://img.shields.io/npm/v/gulp-eval.svg?style=flat)](http://www.npmjs.org/package/gulp-eval) [![Build Status](http://img.shields.io/travis/gulp-bem/gulp-eval/master.svg?style=flat&label=tests)](https://travis-ci.org/gulp-bem/gulp-eval) [![Coverage Status](https://img.shields.io/coveralls/gulp-bem/gulp-eval.svg?branch=master&style=flat)](https://coveralls.io/r/gulp-bem/gulp-eval) [![Dependency Status](http://img.shields.io/david/gulp-bem/gulp-eval.svg?style=flat)](https://david-dm.org/gulp-bem/gulp-eval)
+
+Require commonJS modules, json files or eval expression inside gulp tasks with ease.
+
+gulp-eval fills `data` property of [Vinyl](https://github.com/gulpjs/vinyl) files.
+
+## Example of use:
+```js
+var gulp = require('gulp');
+var concat = require('gulp-concat');
+var gulpEval = require('gulp-eval');
+var through = require('through2');
+
+gulp.task('default', () => {
+  return gulp.src('node_modules/*/package.json')
+    .pipe(gulpEval())
+    .pipe(through.obj(function(file, enc, next) {
+      var data = file.data; // Here we have evaluated package.json
+      file.contents = new Buffer(data.name + '#' + data.version);
+      next(null, file);
+    }))
+    .pipe(concat('all.txt'))
+    .pipe(gulp.dest('dest/'));
+});
+```
+
+## Eval
+You can use eval function without gulp
+
+```js
+var saveEval = require('gulp-eval').eval;
+
+var requireContent =
+`
+    var p = require('../package.json');
+    module.exports = {
+        elem: p.name,
+    };
+`;
+
+var block = saveEval(requireContent);
+
+console.log(block.elem);
+```
+
+## Context
+You can provide some like-a-global variables into eval
```

Comment:
That's cool, but the first purpose is using inside pipe, do we have option to pass context?